batavia/types/Tuple.js

```diff
@@ -547,12 +551,20 @@ Tuple.prototype.count = function(value) {
     return count
 }
 
-Tuple.prototype.index = function(value, start, stop) {
-    if (arguments.length < 1) {
-        throw new exceptions.TypeError.$pyclass('index() takes at least 1 argument (' + arguments.length + ' given)')
-    } else if (arguments.length > 3) {
-        throw new exceptions.TypeError.$pyclass('index() takes at most 3 arguments (' + arguments.length + ' given)')
+Tuple.prototype.count.$pyargs = true
+
+Tuple.prototype.index = function(args, kwargs) {
+    if (args.length < 1) {
+        throw new exceptions.TypeError.$pyclass('index() takes at least 1 argument (' + args.length + ' given)')
+    } else if (args.length > 3) {
+        throw new exceptions.TypeError.$pyclass('index() takes at most 3 arguments (' + args.length + ' given)')
     }
+    if (kwargs && Object.keys(kwargs).length > 0) {
+        throw new exceptions.TypeError.$pyclass('index() takes no keyword arguments')
+    }
+    var value = args[0]
+    if (args[1]) { var start = args[1] }
```

Comment:
The linter apparently doesn't catch this, but single-line if statements aren't preferred style.